Products/ZenUtils/elastic/client.py

```diff
@@ -38,6 +38,12 @@ def __init__(self):
         used across all requests that the client makes.
         """
         self.session = None
+        self.cc_version = ""
+        self._hothOrNewer = checkHothOrNewer()
```

Comment:
With your latest change, checkHothOrNewer returns a tuple which will make self._hothOrNewer evaluate to true since it will not be empty